### ISSUE TITLE
stopped working on any version beyond n8n v1.46.0

### ISSUE THREAD

**Issue by anantanandgupta** [LucasSovre/listmonk-n8n#10]
Hi @LucasSovre, are you planning to maintain this project in future. we have updated to version 1.46.0 and it stopped working. i guess the n8n APIs are changed from this version and the module you have is not able to work with the same. older versions it works fine.  all the requests where the POST/PUT methods are getting called are failing as it is not able to parse the body as per my understanding.

**Comment by LucasSovre** [comment]
Hi @anantanandgupt, i have seen your issue and topic on n8n community forum (https://community.n8n.io/t/community-node-is-no-longer-visible/46665/9)

indeed, I am really busy at work this days... but I will be back in September, if you want I can give you rights to approve merge request, then you will can create release and the GitHub action will take care for pushing new versions to npm registry

**Comment by anantanandgupta** [comment]
thanks @LucasSovre ... i will help you maintain this repository.

**Comment by anantanandgupta** [comment]
I tried to figure out the reason why it is not working on latest versions of n8n . It seems it is something to do with the node defaults. Will investigate more and keep all posted on the findings. It is unfortunate that n8n team is not giving any justifications rather and we don’t have any optimal debugging mechanism where we can test debug the community nodes against different versions of n8n.

**Comment by LucasSovre** [comment]
Hi @anantanandgupta, i will do complete rework of the node in September, using the programmatic node style. I wish it will be easier to maintain and be more compatible with n8n.

**Comment by anantanandgupta** [comment]
> Hi @anantanandgupta, i will do complete rework of the node in September, using the programmatic node style. I wish it will be easier to maintain and be more compatible with n8n.

No point in writing the complex node style. Just need to figure out what causing the post body to become empty in any operation which is calling the post apis.

**Comment by LucasSovre** [comment]
Did you tried to spy the network requests using a tool like wireshark to debug it ?

I wrote a tutorial about it while creating this node : https://www.entrecompetents.fr/blog/en/dev/debug-api-wireshark/

**Comment by anantanandgupta** [comment]
Yes ... Thats how I know that the request body is becoming empty before it is sent. There is somewhere the code overwriting the body after the operation is evaluated. There seems to be a node default configuration which when loaded, overwriting the evaluated operation. Like while iterating the operations and building the request configuration it builds the proper body, but by the end once the iteration is completed, it is set to undefined for body field of the request configuration.

**Comment by AlejandroAkbal** [comment]
Any chance this node starts working again? :(

**Comment by LucasSovre** [comment]
yes, I planed to work on this but a little busy, promise I will take a look at fixing this

**Comment by NeZios** [comment]
Same issue here. Only POST/PUT methods are not working. 
Thanks in advance!

**Comment by NeZios** [comment]
Hi ! Any news on updating this n8n node ? 
Thanks!

**Comment by anantanandgupta** [comment]
> Hi ! Any news on updating this n8n node ? 
> Thanks!

see if you can use https://www.npmjs.com/package/n8n-nodes-listmonk-service in the meantime. it has limited methods, but it is working.

let me know if you need any other APIs to be implemented.

**Comment by LucasSovre** [comment]
@NeZios and @anantanandgupta no promise on result, but I have planned to work on this tomorrow !

**Comment by anantanandgupta** [comment]
I am frustrated with the recent breaking changes and inconsistent updates in n8n, especially concerning community-driven development. The tooling for node development has seen minimal updates over the past two years, leading to compatibility issues and unexpected behavior.

For instance, the n8n-nodes-starter project faced build failures due to an unresolved ESLint dependency, as discussed in Issue #4142. Additionally, the upgrade of chokidar to version 4 broke hot-reloading for node development, as noted in Issue #11577.

Moreover, the lack of active support for community developers is evident. Attempts to update or enhance custom nodes often result in challenges during linting or compilation, with limited guidance available. The absence of a dedicated team to address community concerns exacerbates this issue.

The frequent breaking changes without adequate deprecation warnings further complicate development. For example, the Merge node underwent significant changes, including a rearrangement of operations, as detailed in the Breaking Changes documentation. Such modifications can render existing workflows obsolete without prior notice.

Given these challenges, it's becoming increasingly difficult to justify investing time and effort into creating or updating nodes within the n8n ecosystem. The platform's current trajectory suggests a diminishing focus on open-source and community-driven development, which raises concerns about its future viability.

Consequently, I'm considering exploring alternative automation platforms that offer more stability and robust community support. If anyone has recommendations for reliable alternatives, I'd appreciate your insights.

**Comment by NeZios** [comment]
> [@NeZios](https://github.com/NeZios) and [@anantanandgupta](https://github.com/anantanandgupta) no promise on result, but I have planned to work on this tomorrow !

Thank you so much! Keep us in touch !

**Comment by NeZios** [comment]
Any news @LucasSovre ?

**Comment by LucasSovre** [comment]
Yes, I can’t really figure out why is this happening, but in despite of n8n lack of answer and water to debug a node, I wonder about switching to windmill ?

**Comment by anantanandgupta** [comment]
windmill a good choice but the integrationis error pron and lack of reusable workflow pattern. 

> Yes, I can’t really figure out why is this happening, but in despite of n8n lack of answer and water to debug a node, I wonder about switching to windmill ?
>
